cli/testflinger_cli/__init__.py

```diff
@@ -510,6 +511,12 @@ def jobs(self):
 
     def list_queues(self):
         """List the advertised queues on the current Testflinger server"""
+        print(
+            "ATTENTION: This only shows a curated list of queues with "
+            "descriptions, not ALL queues. If you can't find the queue you "
+            "want to use, a job can still be submitted for queues not listed "
+            "here.\n"
+        )
```

Comment:
I had considered whether to move it to a function to avoid the duplication, but it seemed a bit overkill for a single print statement. Happy to do that if you prefer, but we can't just put it in _get_image(), because list_queues() doesn't call that

Reply:
We could have the warning string stored somewhere

Reply:
Not sure what you mean exactly, but I moved it to a helper function so it's not duplicated. Better?